backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -134,18 +135,16 @@ public Optional<User> getUserById(String id) {
 		return userRepository.findById(id).map(userMapper::toUser);
 	}
 
-	public void updateUser(String userId, String email) {
+	public void updateUser(String userId, UserPatch userPatched) {
 		Assert.hasText(userId, "userId is required; it must not be null or blank");
-		Assert.hasText(email, "email is required; it must not be null or blank");
+		Assert.notNull(userPatched, "patchedUser is required; it must not be null");
```

Comment:
```suggestion
	public void updateUser(String userId, UserPatch userPatch) {
		Assert.hasText(userId, "userId is required; it must not be null or blank");
		Assert.notNull(userPatch, "userPatch is required; it must not be null");
```

Reply:
In case it isn't clear.. The `Assert.notNull(userPatched, "patchedUser is required; it must not be null")` line had a typo or copy/paste error.. while suggesting a fix for that I also renamed the variable from `userPatched` to `userPatch`.

Reply:
fixed